### PR TITLE
ENGINE, CHARSET, COLLATE Flag Support

### PR DESCRIPTION
**Pull request by akandels**
Adding support for engine, charset and collate options at the CREATE.

### CODE REVIEW

**Comment by wzrdtales** [comment]
Thanks for your contribution! :)

To merge this a test to cover this is needed also.

File: index.js [hunk 1]
```diff
@@ -277,6 +277,81 @@ var MysqlDriver = Base.extend({
...
+      delete options.columns;
+      tableOptions = options;
+    }
+
+    var ifNotExistsSql = "";
+    if(tableOptions.ifNotExists) {
+      ifNotExistsSql = "IF NOT EXISTS";
+    }
+
+    var engineSql = '';
```
Comment (wzrdtales) [inline index.js]: May be should move that alltogether in one string, or do you see any reason to keep it like that. Would not like to have a huge list of parameters afterwards of strings which have no real reason to be sperated though :)

**Comment by akandels** [comment]
Didn't realize how everything connected. I see the tests now in the other repo, working on that.

Yeah we could probably make params an array and join it together, I was mostly just copying the pattern there. Thanks!

File: index.js (hunk 1 above)
Comment (wzrdtales) [inline index.js]: For an example see the cockroachdb driver I just lately added: https://github.com/db-migrate/cockroachdb/blob/master/index.js

**Comment by wzrdtales** [comment]
This has been added as the feature via the new applyTableOptions.

**Pull request closed without merging**